### PR TITLE
Http ingest

### PR DESCRIPTION
**Pull request by sebest**

### CODE REVIEW

File: sse.go [hunk 1]
```diff
@@ -91,8 +102,47 @@ func (daemon *SSEDaemon) Status(w http.ResponseWriter, r *http.Request) {
...
+
+	if r.Header.Get("Content-Type") != "application/json" {
+		w.WriteHeader(415)
+		return
+	}
+
+	h := w.Header()
+	h.Set("Server", fmt.Sprintf("oplog/%s", Version))
+	h.Set("Cache-Control", "no-cache, no-store, must-revalidate")
+	h.Set("Connection", "close")
```
Comment (rs) [inline sse.go]: You don't want to allow keep-alive?

File: ingest.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+type inOperation struct {
+	Event     string     `json:"event"`
+	Parents   []string   `json:"parents"`
+	Type      string     `json:"type"`
+	ID        string     `json:"id"`
+	Timestamp *time.Time `json:"timestamp,omniempty"`
+}
+
+// ingestOperation parses JSON data and returns an Operation on success.
+func ingestOperation(data []byte) (*Operation, error) {
```
Comment (rs) [inline ingest.go]: Wouldn't decodeOperation a better name?

File: sse.go (hunk 1 above)
Comment (sebest) [inline sse.go]: "close" removed

File: ingest.go (hunk 2 above)
Comment (sebest) [inline ingest.go]: renamed

**Pull request merged**